resources/META-INF/plugin.xml

```diff
@@ -120,6 +120,9 @@
 
   <extensions defaultExtensionNs="com.intellij">
     <macro implementation="com.twitter.intellij.pants.macro.FilePathRelativeToBuiltRootMacro"/>
+    <localInspection bundle="com.twitter.intellij.pants.PantsBundle" key="pants.info.python.facet.missing"
+                     groupName="Pants" enabledByDefault="true"
+                     implementationClass="com.twitter.intellij.pants.inspection.PythonFacetInspection"/>
```

Comment:
Hey might want to move this to `pants-python.xml` otherwise when python plugin is disabled, it will have the following error when clicking `Add python facet`:
![Screen Shot 2020-02-10 at 11 10 31 AM](https://user-images.githubusercontent.com/596358/74181421-27345300-4bf6-11ea-9d1f-47287b57be04.png)